app/models/news/item.rb

```diff
@@ -45,7 +50,7 @@ def validate
 
     dataset_module do
       def descending
-        order(Sequel.desc(:start_date), Sequel.desc(:id))
+        order { [Sequel.desc(news_items[:start_date]), Sequel.desc(news_items[:id])] }
```

Comment:
Is this not the same as before but just passing the configuration to an eval'd block?

Reply:
originally I was running into issues because the first syntax doesn't assign table names where as the second doesn't but looking at it again it does see unneeded so I'll test again and see if it is necessary - was solving this one right at the end of friday so was pretty fried :smile: